Libs/MRML/Logic/vtkMRMLApplicationLogic.cxx

```diff
@@ -312,6 +312,93 @@ void vtkMRMLApplicationLogic::PropagateVolumeSelection(int fit)
     }
 }
 
+//----------------------------------------------------------------------------
+void vtkMRMLApplicationLogic::PropagateBackgroundVolumeSelection(int fit)
```

Comment:
Steve, I thought when we talked, you gave strong preference to separate methods instead of using enum, which I initially suggested.... 

It's funny, Alireza actually implemented the enum approach first, but I asked him to switch to separate methods, because I thought that's what you wanted. A lot of confusion....